### PR TITLE
Rename techstoreclub-simple-comic.rb to simple-comic.rb

### PR DESCRIPTION
**Pull request by vg**
_If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so._

After making all changes to the cask:
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:
- [x] Named the cask according to the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked that the cask was not already refused in [closed issues](https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed).

Reason: Previous Cask name was a fork of the original app

### CODE REVIEW

**Comment by vitorgalvao** [comment]
> Reason: Previous Cask name was a fork of the original app

[There’s a reason for that](https://github.com/caskroom/homebrew-cask/pull/10989). That said, it appears to have been resolved.

However, `checkpoint:` seems to be wrong.

**Comment by jawshooah** [comment]
> That said, it appears to have been resolved.

How so?

**Comment by vitorgalvao** [comment]
> How so?

The url is no longer dead.

The fork only existed to provide a working download.

**Comment by jawshooah** [comment]
Really? This is what I see when I click "Download":

```
This XML file does not appear to have any style information associated with it. The document tree is shown below.
<Error>
<Code>AllAccessDisabled</Code>
<Message>All access to this object has been disabled</Message>
<RequestId>7F8B17E00596AA09</RequestId>
<HostId>
RR14r3wYcAOZ28RMfhaBRnq1Wrf9PeKptp0Znwkj1k04DbKQlcgy65ZopJZ+r//Ag9c/barhqf0=
</HostId>
</Error>
```

**Comment by vitorgalvao** [comment]
Just tried it again, and it downloaded fine. The download link in the page points to a github releases download, and [it does exist](https://github.com/dancingtortoise/Simple-Comic/releases).

File: Casks/simple-comic.rb [hunk 1]
```diff
@@ -2,6 +2,7 @@
   version '1.7_252'
   sha256 '4ddd18a02a79fc8201824e6ab99291c6d4c8680f79f94bc372bf71f0535def35'
 
+  # dancingtortoise.github.io was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/simple-comic.rb]: It’s supposed to be `github.com/dancingtortoise/Simple-Comic`.

File: Casks/simple-comic.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+cask 'simple-comic' do
+  version '1.7_252'
+  sha256 '4ddd18a02a79fc8201824e6ab99291c6d4c8680f79f94bc372bf71f0535def35'
+
+  # dancingtortoise.github.io was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/simple-comic.rb]: It’s supposed to be `github.com/dancingtortoise/Simple-Comic`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/simple-comic.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+cask 'simple-comic' do
+  version '1.7_252'
+  sha256 '4ddd18a02a79fc8201824e6ab99291c6d4c8680f79f94bc372bf71f0535def35'
+
+  url "https://github.com/dancingtortoise/Simple-Comic/releases/download/#{version}/SimpleComic_#{version}.zip"
+  appcast 'https://github.com/dancingtortoise/Simple-Comic/releases.atom',
+          checkpoint: '66df09ef39345ef6782eb01eee3cc86d8d2a9a7860d963f55a88b62cd143b976'
+  name 'Simple Comic'
+  homepage 'https://github.com/dancingtortoise/Simple-Comic'
```
Comment (vitorgalvao) [inline Casks/simple-comic.rb]: No, the page was correct. The comment was wrong.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/simple-comic.rb (hunk 3 above)
Comment (vg) [inline Casks/simple-comic.rb]: Sorry I am new to this. What was exactly wrong in the comment.

File: Casks/simple-comic.rb (hunk 2 above)
Comment (vg) [inline Casks/simple-comic.rb]: Done.

File: Casks/simple-comic.rb (hunk 3 above)
Comment (vitorgalvao) [inline Casks/simple-comic.rb]: The comment referenced the homepage, when it should have referenced the download url. I’ll make the fix, so you can see.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**